TomiwaSos/GardeningApp
# Initial CRUD for clients

This it a good template on how we should be making the CRUDS for our models, feel free to use these documents as a tempale.

Everything has been tested using psotman and you should do the same. Download the app directly as this will allow you to test localhost.

Instructions on how to test using postman:

1. Run rails s
2. open postman
3. run a command
    - Make sure you are using accurate commands, which look like as follows 
![image](https://github.com/TomiwaSos/GardeningApp/assets/86957496/155e4cc2-8d5a-45b8-970f-f710d5e2cd21)
    - an example url will be 127.0.0.1:3000/clients?name=tomiwa&address=house&number=07718951442&email=tomiwa@doyoucustoms.com
    - and make sure this is as a post
